tl/console-and-shells/completion-shell.rst

```diff
@@ -1,91 +1,91 @@
-Completion Shell
+Pagkumpleto ng Shell
 ################
```

Comment:
Title underlines need to match the title length. For example this should be:

```rst
Pagkumpleto ng Shell
####################
```

When title underlines are too short, sphinx fails to create headlines correctly.